lib/swoosh/adapters/smtp/helpers.ex

```diff
@@ -66,6 +66,17 @@ if Code.ensure_loaded?(:mimemail) do
       Map.to_list(additional_headers) ++ headers
     end
 
+    defp prepare_parts(headers, %Email{
```

Comment:
```
Delivered-To: chenpaul914@gmail.com
Received: by 10.182.149.2 with SMTP id tw2csp1117181obb;
        Sun, 19 Mar 2017 15:54:10 -0700 (PDT)
X-Received: by 10.98.78.201 with SMTP id c192mr29468577pfb.246.1489964050679;
        Sun, 19 Mar 2017 15:54:10 -0700 (PDT)
Return-Path: <msprvs1=17251O0p3hCxr=bounces-73381@mail.primeminddigital.com>
Received: from mta200.spmta.com (mta200.spmta.com. [35.160.4.15])
        by mx.google.com with ESMTPS id y9si15429616pli.39.2017.03.19.15.54.10
        for <chenpaul914@gmail.com>
        (version=TLS1_2 cipher=ECDHE-RSA-AES128-GCM-SHA256 bits=128/128);
        Sun, 19 Mar 2017 15:54:10 -0700 (PDT)
Received-SPF: pass (google.com: domain of msprvs1=17251o0p3hcxr=bounces-73381@mail.primeminddigital.com designates 35.160.4.15 as permitted sender) client-ip=35.160.4.15;
Authentication-Results: mx.google.com;
       dkim=pass header.i=@primeminddigital.com;
       spf=pass (google.com: domain of msprvs1=17251o0p3hcxr=bounces-73381@mail.primeminddigital.com designates 35.160.4.15 as permitted sender) smtp.mailfrom=msprvs1=17251O0p3hCxr=bounces-73381@mail.primeminddigital.com
Return-Path: <po@primeminddigital.com>
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=primeminddigital.com; s=scph0317; t=1489964050; i=@primeminddigital.com; bh=jV37ACoBXM+4hZ2Dr4BXMlx9QMAsq4v2MlNDwpp8AcU=; h=From:To:Subject:Date:List-Unsubscribe:List-Id; b=km8DNmJRrn/+SDD8CNL6tuqDd1JzGWXdUExURtdMMqgW4uyH7C4z1t2ZwBVpYYDnV
	 50/Wtg+AdwaZXqeyUhw+wNjpz/eHP8XWoYP0998LAOCkSOrCu+wCs6ehRUzumBSPtz
	 MpvTLWRjkoc6ViLRVqjxE6kZm3auizE8RKX4jiVU=
X-MSFBL: mOpLRid5gi+zkzrmr5ubfXYqbnDiZf3GF5wtyXURxfs=|eyJiIjoiaXBfMzUuMTY wLjQuMTUiLCJzZW5kaW5nX2lwIjoiMzUuMTYwLjQuMTUiLCJ0ZW1wbGF0ZV92ZXJ zaW9uIjoiMCIsImciOiJiZ19oZWxsb18xIiwiZnJpZW5kbHlfZnJvbSI6InBvQHB yaW1lbWluZGRpZ2l0YWwuY29tIiwicmNwdF90YWdzIjpbIF0sInJjcHRfbWV0YSI 6eyB9LCJ0ZW1wbGF0ZV9pZCI6InNtdHBfMTI0OTU3NTczODkxMjIwMTIiLCJyIjo iY2hlbnBhdWw5MTRAZ21haWwuY29tIiwidGVuYW50X2lkIjoic3BjIiwiaXBfcG9 vbCI6InNoYXJlZCIsImN1c3RvbWVyX2lkIjoiNzMzODEiLCJ0cmFuc21pc3Npb25 faWQiOiIxMjQ5NTc1NzM4OTEyMjAxMiIsInN1YmFjY291bnRfaWQiOiIwIiwibWV zc2FnZV9pZCI6IjAwMDAxMTBjY2Y1OGI5N2ZhMjE5IiwiaXBfcG9vbF9yYXciOiJ oZWxsb18xIn0=
Authentication-Results: c.mta1vsmtp.cc.prd.sparkpost smtp.user=smtp_injection; auth=pass (LOGIN)
Received: from [120.144.136.45] ([120.144.136.45:65156] helo=DESKTOP-U92GK7C) by c.mta1vsmtp.cc.prd.sparkpost (envelope-from <po@primeminddigital.com>) (ecelerity 4.2.30.59184 r(Core:4.2.30.5)) with ESMTPSA (cipher=ECDHE-RSA-AES256-GCM-SHA384) id 2A/91-32697-11C0FC85; Sun, 19 Mar 2017 22:54:10 +0000
From: po@primeminddigital.com
To: chenpaul914@gmail.com
Subject: test smtp
Mime-Version: 1.0
Content-Type: multipart/mixed; boundary="_=0g570n704241404c684s4m3d4m4f693r=_"
Date: Mon, 20 Mar 2017 08:54:05 +1000
Message-ID: <033ece5afd0ef43dfca028766fcc5212@DESKTOP-U92GK7C>
List-Unsubscribe: <mailto:unsubscribe@unsub.spmta.com?subject=unsubscribe:dZeBh2VgW4a1_UsFVRXkemW89703onAMYM99ZubaJeE~|eyAicmNwdF90byI6ICJjaGVucGF1bDkxNEBnbWFpbC5jb20iLCAicmNwdF9tZXRhIjogInt9IiwgInJjcHRfdGFncyI6ICJbIF0iLCAidGVuYW50X2lkIjogInNwYyIsICJ0ZW1wbGF0ZV9pZCI6ICJzbXRwXzEyNDk1NzU3Mzg5MTIyMDEyIiwgInRyYW5zbWlzc2lvbl9pZCI6ICIxMjQ5NTc1NzM4OTEyMjAxMiIsICJzZW5kaW5nX2lwIjogIjM1LjE2MC40LjE1IiwgInRlbXBsYXRlX3ZlcnNpb24iOiAiMCIsICJtYWlsZnJvbSI6ICJwb0BwcmltZW1pbmRkaWdpdGFsLmNvbSIsICJpcF9wb29sIjogInNoYXJlZCIsICJpcF9wb29sX3JhdyI6ICJoZWxsb18xIiwgImN1c3RvbWVyX2lkIjogIjczMzgxIiwgInN1YmFjY291bnRfaWQiOiAiMCIsICJtZXNzYWdlX2lkIjogIjAwMDAxMTBjY2Y1OGI5N2ZhMjE5IiwgImZyaWVuZGx5X2Zyb20iOiAicG9AcHJpbWVtaW5kZGlnaXRhbC5jb20iIH0~>
List-Id: <spc-73381-0>

--_=0g570n704241404c684s4m3d4m4f693r=_
Content-Type: text/plain;
	 charset="utf-8"
Content-Transfer-Encoding: quoted-printable
Content-Disposition: inline

hi yo
--_=0g570n704241404c684s4m3d4m4f693r=_
Content-Transfer-Encoding: base64
Content-Type: application/octet-stream
Content-Disposition: attachment; filename=README.md

SXlCVVpYTjBVM0FLQ2lvcVZFOUVUem9nUVdSa0lHUmxjMk55YVhCMGFXOXVLaW9LQ2lNaklFbHVj
M1JoYkd4aGRHbHZiZ29LU1dZZ1cyRjJZV2xzWVdKc1pTQnBiaUJJWlhoZEtHaDBkSEJ6T2k4dmFH
VjRMbkJ0TDJSdlkzTXZjSFZpYkdsemFDa3NJSFJvWlNCd1lXTnJZV2RsSUdOaGJpQmlaU0JwYm5O
MFlXeHNaV1FLWW5rZ1lXUmthVzVuSUdCMFpYTjBYM053WUNCMGJ5QjViM1Z5SUd4cGMzUWdiMlln
WkdWd1pXNWtaVzVqYVdWeklHbHVJR0J0YVhndVpYaHpZRG9LQ21CZ1lHVnNhWGhwY2dwa1pXWWda
R1Z3Y3lCa2J3b2dJRnQ3T25SbGMzUmZjM0FzSUNKK1BpQXdMakV1TUNKOVhRcGxibVFLWUdCZ0Nn
cEViMk4xYldWdWRHRjBhVzl1SUdOaGJpQmlaU0JuWlc1bGNtRjBaV1FnZDJsMGFDQmJSWGhFYjJO
ZEtHaDBkSEJ6T2k4dloybDBhSFZpTG1OdmJTOWxiR2w0YVhJdGJHRnVaeTlsZUY5a2IyTXBDbUZ1
WkNCd2RXSnNhWE5vWldRZ2IyNGdXMGhsZUVSdlkzTmRLR2gwZEhCek9pOHZhR1Y0Wkc5amN5NXdi
U2t1SUU5dVkyVWdjSFZpYkdsemFHVmtMQ0IwYUdVZ1pHOWpjeUJqWVc0S1ltVWdabTkxYm1RZ1lY
UWdXMmgwZEhCek9pOHZhR1Y0Wkc5amN5NXdiUzkwWlhOMFgzTndYU2hvZEhSd2N6b3ZMMmhsZUdS
dlkzTXVjRzB2ZEdWemRGOXpjQ2t1Q2dvPQ==
--_=0g570n704241404c684s4m3d4m4f693r=_--
```
This is one of the test emails I sent to myself with Swoosh SMTP, it seems to handle that part already?